projects/plugins/boost/app/assets/src/js/features/cornerstone-pages/cornerstone-pages.tsx

```diff
@@ -37,6 +38,9 @@ const CornerstonePages = () => {
 					<PanelRow>
 						<Meta />
 					</PanelRow>
+					<PanelRow>
```

Comment:
As discussed on the call, let's hide it completely for WP < 6.8

Reply:
Done. The toggle is entirely hidden now by checking the WP version in PHP.